src/event-bridge/event-bus.ts

```diff
@@ -327,3 +355,44 @@ class ImportedEventBus<E extends EventBusRuleInput> extends EventBusBase<E> {
     super(bus);
   }
 }
+
+export interface EventBusTargetIntegration<
```

Comment:
Should we document these interfaces? Are they public interfaces?

Reply:
Errr. they are to integrators. I'll add some docs.